### PR TITLE
Dictionary update

### PR DESCRIPTION
**Pull request by Jean-Du**
Updating the dictionary and unit conversion to accommodate the output of new TopoStats. Fixing the lack of brackets in some printing statements.

### CODE REVIEW

File: Plotting.py
Comment (MaxGamill-Sheffield) [inline Plotting.py:478]: Should be "Path to the csv file".

File: Plotting.py
Comment (MaxGamill-Sheffield) [inline Plotting.py:480]: Perhaps add an example path like "./output/all_statistics.csv"?

**Review by MaxGamill-Sheffield — APPROVED** [review]
Works for me and produces both the area KDE plot and contour length histogram when pointing to `all_statistics.csv`.

Although this entire module needs to be refactored and become pip installable with the rest of TopoStats, this is a very good current solution to generate plots from the newer output files.

A few very minor changes changes but this will all change when refactoring takes place.

**Comment by SylviaWhittle** [comment]
Hi @Jean-Du, I have some changes/suggestions that I would like to request - small edits like bringing in the use of `pathlib` for example. Could you hold off on merging for a little bit?

**Comment by SylviaWhittle** [comment]
@Jean-Du I have made some edits in `SylviaWhittle/plotting-edits` that might be of interest to you. I could make a PR against this branch, or I could make a PR against `dev` after this is merged. What do you think the best course would be @ns-rse? I'm leaning towards the latter but would like to know your thoughts on best practice.

File: Plotting.py
Comment (Jean-Du) [reply Plotting.py:480]: @SylviaWhittle has noticed issues with using relative paths and is working on adding pathlib. It might be better to have her add that example path in her new pull request?

**Comment by Jean-Du** [comment]
> Very good, works for me and I concur with the comments that have been made previously.
> 
> I have some edits in https://github.com/AFM-SPM/TopoStats/tree/SylviaWhittle/plotting-edits that might be useful, however I think I shall make a separate PR for those against `dev`

@SylviaWhittle Thank you very much for doing this! Since you'll be making a separate PR, would it be ok for me to merge this one for now?

**Comment by ns-rse** [comment]
Sorry for getting back so late to this, generally the best approach is to merge into `dev` and resolve any conflicts. As you say @SylviaWhittle the whole module needs refactoring and moving under `topostats` so that it becomes part of the installation process.

**Pull request merged**
